Docs/source/developers/testing.rst

```diff
@@ -47,97 +30,149 @@ You can run those locally before you push to save some time, install them once l
 
 See `pre-commit.com <https://pre-commit.com>`__ and our ``.pre-commit-config.yaml`` file in the repository for more details.
 
+How to run automated tests locally
+----------------------------------
+
+Once your new feature is ready, there are ways to check that you did not break anything.
+WarpX has automated tests running every time a commit is pushed to an open pull request.
+The input files and scripts used by the automated tests can be found in the `Examples <https://github.com/ECP-WarpX/WarpX/tree/development/Examples>`__ directory, either under `Physics_applications <https://github.com/ECP-WarpX/WarpX/tree/development/Examples/Physics_applications>`__ or `Tests <https://github.com/ECP-WarpX/WarpX/tree/development/Examples/Tests>`__.
+
+For easier debugging, it can be convenient to run the tests on your local machine by executing CTest as illustrated in the examples below (where we assume that WarpX was configured and built in the directory ``build``):
+
+* List tests available for the current build options:
+
+  .. code-block:: sh
+
+        ctest --test-dir build -N
```

Comment:
I think these are overindented blocks.

Reply:
Done in [cb5e42c](https://github.com/ECP-WarpX/WarpX/pull/5068/commits/cb5e42cf8e2102a3040dea7032404911fe17ef5b).